tools/deployment/make_osx_package.sh

```diff
@@ -0,0 +1,373 @@
+#!/usr/bin/env bash
+
+#  Copyright (c) 2014-present, Facebook, Inc.
+#  All rights reserved.
+#
+#  This source code is licensed in accordance with the terms specified in
+#  the LICENSE file found in the root directory of this source tree.
+
```

Comment:
Since the script most likely doesn't work, is it possible to add a warning here instead of just reverting the file? Something like a `echo "Not supported! Please read the build guide at XXXX"`

Reply:
Totally, I will follow up and make this change in an upcoming PR. I would also like to fix these scripts so they work again.